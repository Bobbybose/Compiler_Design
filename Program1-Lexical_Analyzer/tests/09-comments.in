"Before comments"

// This is a comment
/* Block comment */
/* Block
comment
on
multiple
lines
*/

</*comment*/=

"After comments"

