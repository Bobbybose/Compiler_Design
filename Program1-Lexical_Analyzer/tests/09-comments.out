string-literal: "Before comments": 1, 1
<: 12, 1
=: 12, 13
string-literal: "After comments": 14, 1